client/lib/cart-values/index.js

```diff
@@ -16,6 +16,10 @@ function emptyCart( siteID ) {
 	return { blog_id: siteID, products: [] };
 }
 
+function emptyTemporaryCart( siteID ) {
```

Comment:
@Tug Before we set the temporary flag, was the user's cart cleared when starting a free trial?

What do you think about either calling `emptyCart` from this method, or extending it to use an optional `attributes` object?

```js
function emptyCart( siteId, attributes ) {
  return Object.assign( { blog_id: siteId, products: [], attributes } );
}

var cart = emptyCart( { temporary: true } );
```